topics/ingest-feedbacks.yaml

```diff
@@ -0,0 +1,17 @@
+topic: ingest-feedbacks
+pipeline: replays
+description: User feedback widget events
+services:
+  producers:
+    - getsentry/sentry
+  consumers:
+    - getsentry/sentry
```

Comment:
just out of curiosity, why does this moving to Kafka at all if Sentry is both producer and consumer and it is low volume? Our existing user feedback product is not a Kafka pipeline.

Reply:
made a mistake, producer is relay